docs/knowledgebase/getting-started/glossary.md

```diff
@@ -2,38 +2,39 @@
 title: Glossary
 ---
 
-Glossary (alphabetical) of terms and lingo used in the Substrate and Polkadot codebases, as well as
-architecture.
+This page defines and explains many of the domain-specific terms that are common to the Substrate
+ecosystem. This is a helpful resource for even the most experienced Substrate developers.
 
-## Author (aka Block Author, Block Producer)
-
-The node, actor or identity that is responsible for the creation of a block. Synonymous with "miner"
-or "coinbase" on legacy PoW chains.
+---
 
 ## Adaptive Quorum Biasing (AQB)
 
-A means of specifying the quorum needed for a proposal to pass dependent upon voter turnout.
-Positive bias means requiring increasingly more aye votes than nay votes as there is lower turnout;
-negative (aka inverted) bias is the opposite (see the
+A means of specifying a passing threshold based on the quorum (minimum vote) needed for a proposal
+to pass dependent upon voter turnout. Positive bias means requiring increasingly more aye votes than
+nay votes as there is lower turnout; negative (aka inverted) bias is the opposite (see the
 [graphs](https://youtu.be/VsZuDJMmVPY?t=25413)). It avoids the need for strict quorums, which are
-arbitrary and bring about undesirable mechanics.
+arbitrary and bring about undesirable mechanics. Adaptive quorum biasing is used by the
+[FRAME Democracy pallet](../runtime/frame#democracy), which is used by chain-governing
+[councils](#council) on a number of Substrate-based chains.
```

Comment:
I feel like you are working against the meanings of the words. E.g. "As there is lower turnout": turnout only increases, people can change their votes, or even withdraw them, but they can't "un-show up".

A means of specifying a passing threshold for a referendum based on voter turnout. Positive turnout bias means that as more stake votes, the passing threshold decreases; i.e. a higher turnout _positively_ increases the likelihood of a referendum passing. Negative turnout bias means that as more stake votes, the passing threshold increases. Negative turnout bias is also called "default carries" as, given an apathetic voting body, the referendum will pass by default. A neutral turnout bias is simply a majority carries vote. AQB removes the need for strict quorums. AQB is implemented in the Democracy pallet, which provides the interfaces for other on-chain bodies (e.g. a Collective or any token holder) to call referenda with positive, neutral, or negative biases.

Reply:
I made a few tweaks let...hope I didn't go too far.